scene/gui/line_edit.h

```diff
@@ -100,7 +100,7 @@ class LineEdit : public Control {
 
 	int caret_column = 0;
 	int scroll_offset = 0;
-	int max_length = 0; // 0 for no maximum.
+	int max_length = 10000; // 10,000 so it's not too big by default
```

Comment:
I would bump it to something bigger, like 64 K.

Selection performance, which was the main issue cause, should be fixed by #60136.

Reply:
> I would bump it to something bigger, like 64 K.

Really? **_10,000_** characters for a LineEdit already seemed very generous. I can't imagine there are many use cases where you'd ever need more than 10,000 characters visible in a single LineEdit, to begin with. This is simply its default setting, if the user **_really_** needs more than 10,000 characters, they can set it themself.

Reply:
Max. path length can be `32767` for example, and `LineEdit` is used by all file selectors. But this is just a suggesting anyway.